src/oop/programs_with_multiple_classes/hard/q3/Book.java

```diff
@@ -0,0 +1,15 @@
+package oop.programs_with_multiple_classes.hard.q3;
+
+public class Book {
```

Comment:
Looking back at the medium category, we made this category introduce the concept that objects can have other objects in them, ie for this question, the bookcase class will have an array of books. Can you update this question and the others you are working on in this category like this?